### PR TITLE
Apply black Bio/Seq*.py

### PR DESCRIPTION
**Pull request by peterjc**
Manually fixed implicit string concatenation after black

This pull request addresses issue #2008

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

**Comment by peterjc** [comment]
@svalqui would you like to review this?

File: Bio/Seq.py [hunk 1]
```diff
@@ -1976,11 +2031,13 @@ def __init__(self, data, alphabet=Alphabet.generic_alphabet):
...
             self.data = array.array(self.array_indicator, data)
         elif isinstance(data, (Seq, int, float)):
-            raise TypeError("The sequence data given to a MutableSeq object "
-                            "should be a string or an array "
-                            "(not a Seq object etc)")
+            raise TypeError(
+                "The sequence data given to a MutableSeq object "
+                "should be a string or an array "
+                "(not a Seq object etc)"
+            )
```
Comment (svalqui) [inline Bio/Seq.py]: The last 2 strings should fit in one line.

**Comment by svalqui** [comment]
I only found on thing to comment the rest seems fine to me.

File: Bio/Seq.py (hunk 1 above)
Comment (peterjc) [reply Bio/Seq.py]: Yes - you have a good eye for these.

**Pull request merged**